configcatclient/localdictionarydatasource.py

```diff
@@ -23,8 +30,12 @@ def __init__(self, source, override_behaviour, log):
                 value_type = STRING_VALUE
             elif isinstance(value, int):
                 value_type = INT_VALUE
-            else:
+            elif isinstance(value, float):
                 value_type = DOUBLE_VALUE
+            else:
+                value_type = UNSUPPORTED_VALUE
+                self.log.error("Error occurred while reading dictionary: "
+                               "Unsupported value type: %s." % type(value))
```

Comment:
This is a non-agreed log event. Other SDKs don't log such an error message currently. So, I suggest removing this or at least changing it to a debug level event until we standardize the behavior in this case.